test/e2e/util.go

```diff
@@ -253,3 +255,77 @@ func probe(timeout, period, success, failure int) *corev1.Probe {
 		FailureThreshold: int32(failure),
 	}
 }
+
+// updateIngressControllerSpecWithRetryOnConflict gets a fresh copy of
+// the named ingresscontroller, calls mutateSpecFn() where callers can
+// modify fields of the spec, and then updates the ingresscontroller
+// object. If there is a conflict error on update then the complete
+// sequence of get, mutate, and update is retried until timeout is
+// reached.
+func updateIngressControllerSpecWithRetryOnConflict(t *testing.T, name types.NamespacedName, timeout time.Duration, mutateSpecFn func(*operatorv1.IngressControllerSpec)) error {
+	ic := operatorv1.IngressController{}
+	return wait.PollImmediate(1*time.Second, timeout, func() (bool, error) {
+		if err := kclient.Get(context.TODO(), name, &ic); err != nil {
+			t.Logf("error getting ingress %v: %v, retrying...", name, err)
```

Comment:
```suggestion
			t.Logf("error getting ingress controller %v: %v, retrying...", name, err)
```
for consistency and to avoid confusion with `ingresses.config.openshft.io` or `ingresses.networking.k8s.io`.

Reply:
Nice!